gwpy/signal/tests/test_filter_design.py

```diff
@@ -18,15 +18,15 @@
 
 """Unit tests for :mod:`gwpy.signal.filter_design`
 """
-
-import pytest
-
 import numpy
+import pytest
 
 from scipy import signal
 
 from astropy import units
 
+from unittest import mock
+
```

Comment:
Pedantically, since `unittest` is a standard library module, this should be ordered above the others:

```suggestion
from unittest import mock

import numpy

import pytest

from scipy import signal

from astropy import units
```